### PR TITLE
chore(ci): update build variants to use appropriate distros

### PR DESCRIPTION
**Pull request by addaleax**

### CODE REVIEW

**Comment by addaleax** [comment]
Evergreen: https://spruce.mongodb.com/version/64da4dac850e610b5853f33d/tasks

**Comment by addaleax** [comment]
Evergreen: https://spruce.mongodb.com/version/64da572261837d434ce3e96b/tasks

File: .evergreen.yml [hunk 1]
```diff
@@ -10989,9 +10989,14 @@ buildvariants:
       disable_openssl_shared_config_for_bundled_openssl: true
     tasks:
       - name: e2e_tests_linux_x64_700rc10
+  - name: e2e_suse12_x64
+    display_name: "SLES 12 x64 (E2E Tests)"
+    run_on: suse12-sp5-large
+    tasks:
+      - name: e2e_tests_linux_x64
```
Comment (himanshusinghs) [inline .evergreen.yml:10996]: Yikes! Looks like I missed updating the evergreen file in my PR.

**Review by himanshusinghs — APPROVED** [review]
Thank you for this! Looks good 😄

**Pull request merged**